index.js

```diff
@@ -279,6 +288,7 @@ browserslist.parseConfig = env.parseConfig
 browserslist.readConfig = env.readConfig
 browserslist.findConfig = env.findConfig
 browserslist.loadConfig = env.loadConfig
+browserslist.testCanIUse = env.testCanIUse
```

Comment:
Since this method showing warning, it is better to not put it to public API. How will you use this method? ;)

Reply:
true. i would rather do everything in node.js but i saw a loop iterating through caniuse.agents in index.js so i decided to use this loop and not duplicate it in node.js